### DISCUSSION TITLE
datatable filter

### DISCUSSION THREAD

**Discussion by teyyub** [primefaces/community#1584] [PrimeVue]
hi everybody . on my datatable filter could I customize  apply button ?

**Comment by carlosdelros** [comment]
Yes, you can. Take a look at the advanced example, the 3rd column filter (Agent)

https://primevue.org/datatable/#advanced_filter

**Reply by teyyub** [reply]
you are right I checked all code. but what I want is that in there it used calendar but I want to use input mask. in such case I get error getTime . I want to see apply button method and change code according my need
